src/Api/Test/ScriptFileFaker.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace App\Api\Test\fixtures;
```

Comment:
What a strange namespace, it does not fit with the real Path of the file and the last part starts lowercased.
Can't we just move this to a Fixtures subfolder and capitalize the namespace?